api/location/models.py

```diff
@@ -23,15 +23,14 @@ class Location(Base, Utility):
     id = Column(Integer, Sequence('locations_id_seq',
                                   start=1, increment=1), primary_key=True)
     name = Column(String, nullable=False)
-    abbreviation = Column(String, nullable=False)
```

Comment:
Great work @verenceLola, why did you remove this?
same as line 37

Reply:
Kindly have a look at the `background context` of the PR description. I've updated the information.